src/process_utility.c

```diff
@@ -4919,16 +4920,40 @@ process_altertable_reset_options(AlterTableCmd *cmd, Hypertable *ht)
 {
 	List *pg_options = NIL, *compress_options = NIL;
 	List *inpdef = NIL;
+	WithClauseResult *parse_results = NULL;
+
 	/* is this a compress table stmt */
 	Assert(IsA(cmd->def, List));
 	inpdef = (List *) cmd->def;
 	ts_with_clause_filter(inpdef, &compress_options, &pg_options);
 
-	if (compress_options)
+	if (!compress_options)
+		return DDL_CONTINUE;
+
+	parse_results = ts_alter_table_reset_with_clause_parse(compress_options);
+	if (parse_results[AlterTableFlagOrderBy].is_default &&
+		parse_results[AlterTableFlagSegmentBy].is_default)
+	{
 		ereport(ERROR,
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-				 errmsg("columnstore options cannot be reset")));
+				 errmsg("Only compression settings columnstore options can be reset"),
+				 errdetail("Only segmentby and orderby options can be reset")));
```

Comment:
Hmm 
```suggestion
				 errmsg("Only columnstore options can be reset"),
				 errdetail("Only segmentby and orderby options can be reset")));
```
columnstore is the name we want to use consistently